app/js/actions/labOrdersAction.js

```diff
@@ -27,7 +27,7 @@ export const fetchLabOrders = (testOrderType, options) => {
   return ({
     type: FETCH_LAB_ORDERS,
 
-    payload: axiosInstance.get(`order?s=default&totalCount=true&sort=desc&excludeDiscontinueOrders=true&orderTypes=${testOrderType}&activatedOnOrAfterDate=${moment(options.dateFromField).format('YYYY-MM-DD')}&activatedOnOrBeforeDate=${moment(options.dateToField).format('YYYY-MM-DD')}&excludeCanceledAndExpired=${options.excludeCanceledAndExpired}&canceledOrExpiredOnOrBeforeDate=${options.canceledOrExpiredOnOrBeforeDate ? moment(new Date()).format('YYYY-MM-DD') : ''}&fulfillerStatus=${options.fulfillerStatus ? options.fulfillerStatus : ''}&includeNullFulfillerStatus=${includeNullFulfillerStatus !== null ? includeNullFulfillerStatus : ''}&concepts=${options.conceptUuids ? options.conceptUuids : ''}&accessionNumber=${options.accessionNumber}&patient=${options.patient ? options.patient : ''}&v=${ORDER_REP}&limit=${pageSize}&startIndex=${options.page ? options.page * pageSize : 0}`),
+    payload: axiosInstance.get(`order?s=default&totalCount=true&sort=desc&excludeDiscontinueOrders=true&orderTypes=${testOrderType}&activatedOnOrAfterDate=${moment(options.dateFromField).format('YYYY-MM-DD')}&activatedOnOrBeforeDate=${moment(options.dateToField).format('YYYY-MM-DD')}&excludeCanceledAndExpired=${options.excludeCanceledAndExpired}&canceledOrExpiredOnOrBeforeDate=${options.canceledOrExpiredOnOrBeforeDate ? moment(new Date()).format('YYYY-MM-DD') : ''}&fulfillerStatus=${options.fulfillerStatus ? options.fulfillerStatus : ''}&includeNullFulfillerStatus=${includeNullFulfillerStatus !== null ? includeNullFulfillerStatus : ''}&concepts=${options.conceptUuids ? options.conceptUuids : ''}&accessionNumber=${options.accessionNumber ? options.accessionNumber : ''}&patient=${options.patient ? options.patient : ''}&v=${ORDER_REP}&limit=${pageSize}&startIndex=${options.page ? options.page * pageSize : 0}`),
```

Comment:
if options.accessionNumber was undefined here, it was passing in "accessionNumber=undefined, and the search wasn't returning anything.